### ISSUE TITLE
Network Summary updates fail

### ISSUE THREAD

**Issue by 153957** [HiSPARC/publicdb#79]
For some reason the `update_coincidences()` fails.

```
2015-01-16 14:05:26,135 - INFO - [11845] - Processing coincidences and storing ESD for Network Summary: 09 Jan 2015
2015-01-16 14:05:26,141 - INFO - [11846] - Processing coincidences and storing ESD for Network Summary: 08 Jan 2015
2015-01-16 14:06:08,460 - INFO - [11846] - Processing coincidences and storing ESD for Network Summary: 07 Jan 2015
2015-01-16 14:06:08,471 - ERROR - [11696] - Exception occured.
Traceback (most recent call last):
  File "/srv/publicdb/www/scripts/hisparc-update.py", line 42, in <module>
    run()
  File "/srv/publicdb/www/scripts/hisparc-update.py", line 25, in run
    completed = hg.jobs.update_all_histograms()
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 170, in update_all_histograms
    perform_update_tasks()
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 183, in perform_update_tasks
    update_coincidences()
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 237, in update_coincidences
    for network_summary in results:
  File "/srv/publicdb/publicdb_env/lib/python2.7/multiprocessing/pool.py", line 659, in next
    raise value
IndexError: list index out of range
```

**Comment by 153957** [comment]
Perhaps disable Multiprocessing to get a better look at what is happening..

**Comment by 153957** [comment]
Disabled Multiprocessing, this is the full traceback:

```
2015-01-19 09:56:27,501 - INFO - [4020] - Processing coincidences and storing ESD for Network Summary: 08 Jan 2015
2015-01-19 09:56:28,763 - WARNING - [4020] - /srv/publicdb/publicdb_env/lib/python2.7/site-packages/sapphire/clusters.py:616: UserWarning: Could not get info for station 13007, skipping.
  'skipping.' % station, UserWarning)
2015-01-19 09:57:04,234 - ERROR - [4020] - Exception occured.
Traceback (most recent call last):
  File "/srv/publicdb/www/scripts/hisparc-update.py", line 42, in <module>
    run()
  File "/srv/publicdb/www/scripts/hisparc-update.py", line 25, in run
    completed = hg.jobs.update_all_histograms()
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 170, in update_all_histograms
    perform_update_tasks()
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 183, in perform_update_tasks
    update_coincidences()
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 246, in update_coincidences
    network_summary = search_and_store_coincidences_for_network_summary(network_summary)
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 265, in search_and_store_coincidences_for_network_summary
    search_coincidences_and_store_esd(network_summary)
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/jobs.py", line 348, in search_coincidences_and_store_esd
    esd.search_coincidences_and_store_in_esd(network_summary)
  File "/srv/publicdb/www/scripts/../django_publicdb/histograms/esd.py", line 60, in search_coincidences_and_store_in_esd
    coinc.store_coincidences(cluster=cluster)
  File "/srv/publicdb/publicdb_env/lib/python2.7/site-packages/sapphire/analysis/coincidences.py", line 515, in store_coincidences
    self._store_coincidence(coincidence)
  File "/srv/publicdb/publicdb_env/lib/python2.7/site-packages/sapphire/analysis/coincidences.py", line 553, in _store_coincidence
    station_number = self.cluster.stations[station_id].number
IndexError: list index out of range
```

So the problem is probably that the Coincidences tries to get a cluster object for all staitons that had data, however, one station does not have a configuration, causing it to be skipped in sapphire.clusters.HiSPARCStations so the cluster object has less stations than it should, causing the IndexError when the last station (with highest station id) was in a coincidence, because the cluster object has less stations..

This means that all coincidences created since c5f4e6f48334742294c9d884f27a323154c8b15d should be done again after fixing this bug. However, we sometimes have stations that do not yet have a config, hence the error.. Those need to be filtered out from the coincidences or given some default location (0,0) perhaps..

**Comment by 153957** [comment]
Issue is fixed in SAPPHiRE v0.9.30. However some NetworkSummaries need to be updated again.
